CONTRIBUTING.md

```diff
@@ -1,26 +1,90 @@
 # Contributing
 
-## Code of Conduct
+## How to contribute
+
+When contributing to this repository, please first discuss the changes you wish to make through an issue, email, or any other method with the owners of this repository.
 
 All members of the project community must abide by the [SAP Open Source Code of Conduct](https://github.com/SAP/.github/blob/main/CODE_OF_CONDUCT.md).
-Only by respecting each other we can develop a productive, collaborative community.
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting [a project maintainer](.reuse/dep5).
 
-## Engaging in Our Project
+Once you are ready to make a change, please test it appropriately, create a pull request and describe your change in the pull request. The owners of the repository will review your changes as soon as possible.
+
+## Project Structure
+
+This project contains multiple packages, that are managed using pnpm workspaces.
+Productive packages are located in the `packages` directory, test packages are located in the `tests` directory. 
+All dependencies that are used in more than one of the packages are hoisted into the root project.
+Dependencies that occur only once, the binaries will be placed in the `node_modules` of that specific package.
```

Comment:
[q] Is this not commonly done in TS-node projects? Or is this something we choose to do differently in our project?

Reply:
I copied this from the Cloud SDK, but I think now this might be a bit too detailed. There are certain differences in how different package managers resolve dependencies. One line would probably not be enough to explain how pnpm does this. 
I'll just remove it.